hrjobcontract/js/src/job-contract/services/contract-leave.js

```diff
@@ -210,13 +210,14 @@ define([
               return this.leave_amount;
             };
 
-            _.each(leaveType, function (type, typeId) {
+            _.each(leaveType, function (type) {
               modelEntry.leave_type = type.id;
               modelEntry.leave_type_title = type.title;
               modelEntry.leave_calculation_unit_name = type.calculation_unit_name;
               modelEntry.leave_calculation_unit_label = type.calculation_unit_label;
-              modelEntry.leave_amount = 0;
+              modelEntry.leave_amount = parseFloat(type.default_entitlement);
```

Comment:
this changes looks weird. Why was it 0 before? There must have been a reason. And is `parseFloat` necessary at all?

Reply:
You are right, this is unnecessary change. Reverted.